install/kubernetes/cilium/templates/cilium-agent/servicemonitor.yaml

```diff
@@ -5,6 +5,10 @@ kind: ServiceMonitor
 metadata:
   name: cilium-agent
   namespace: {{ .Values.prometheus.serviceMonitor.namespace | default .Release.Namespace }}
+  labels:
+    {{- range $key, $value := .Values.prometheus.serviceMonitor.labels }}
```

Comment:
just a style and mainly for consistency, in other places we are doing something like below.

```
        {{- with .Values.preflight.podLabels }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```

Reply:
I refactored to your suggested syntax.